### PR TITLE
watch list: canceling 'add to new watch list' results in new panel #1519

### LINKED ISSUE #1519: watch list: canceling 'add to new watch list' results in new panel
Steps:
1. Select an issue and choose 'add to watch list --> new watch list'
2. Choose 'cancel' in the confirmation dialog

Expected: no new panel created
Actual: a new panel with the default name is created

### PR DESCRIPTION
**Pull request by raisfathin**
Fixes #1519

### CODE REVIEW

File: src/test/java/unstable/ContextMenuTests.java [hunk 1]
```diff
@@ -169,6 +169,25 @@ public void addToWatchListPanel_createNewWatchListPanel_issueAddedToNewPanel() {
     }
 
     @Test
+    public void addToWatchListPanel_cancelOperation_operationCancelled() {
```
Comment (HansNewbie) [inline src/test/java/unstable/ContextMenuTests.java]: Can we rename the last part to `noNewPanelCreated()`.

File: src/test/java/unstable/ContextMenuTests.java [hunk 2]
```diff
@@ -169,6 +169,25 @@ public void addToWatchListPanel_createNewWatchListPanel_issueAddedToNewPanel() {
...
+        ListPanel listPanel = getPanel(0);
+
```
Comment (HansNewbie) [inline src/test/java/unstable/ContextMenuTests.java]: We should `assert` here that the panel count is 1 in the first place.

**Comment by raisfathin** [comment]
Done!

**Comment by HansNewbie** [comment]
Looks good.

**Pull request merged**